### PR TITLE
swagger API service naming update

### LINKED ISSUE #960: service.swagger.json should have tags specific to the entity
All service.swagger.json have the the tags set to DocumentService. 
The tags is used to generate service clients with a code generator and this means when you a generate a client all methods will go under DocumentServiceApi which will result in method collision:
```
DocumentServiceApi.create() 
DocumentServiceApi.create_1()
DocumentServiceApi.create_2()
```
To fix this we need to define entity specific tags:
1. /invoice/service.swagger.json 
```
"tags": [
   "InvoiceService"
]
```
2. /account/service.swagger.json 
```
"tags": [
   "AccountService"
]
```

### PR DESCRIPTION
**Pull request by charleenfei**
<This is intended as a guide when describing-reviewing pull requests for go-centrifuge>
Close #960.

### CODE REVIEW

File: api/service.go [hunk 1]
```diff
@@ -105,35 +105,35 @@ func registerAPIs(ctx context.Context, cfg Config, InvoiceUnpaidService nft.Invo
 
 func registerDocumentTypes(ctx context.Context, nodeObjReg map[string]interface{}, grpcServer *grpc.Server, gwmux *runtime.ServeMux, addr string, dopts []grpc.DialOption) error {
 	// register invoice
-	invHandler, ok := nodeObjReg[invoice.BootstrappedInvoiceHandler].(invoicepb.DocumentServiceServer)
+	invHandler, ok := nodeObjReg[invoice.BootstrappedInvoiceHandler].(invoicepb.InvoiceServiceServer)
```
Comment (vedhavyas) [inline api/service.go]: I feel `invoicepb.InvoiceService` is redundant but I see this is the only way.

File: protobufs/nft/service.proto [hunk 2]
```diff
@@ -10,7 +10,7 @@ option java_package = "com.nft";
 import "google/api/annotations.proto";
 import "protoc-gen-swagger/options/annotations.proto";
 
-// NFTService let's you mint NFTs for a document
+// // NFTService contains all common interactions for NFTs
```
Comment (vedhavyas) [inline protobufs/nft/service.proto]:
```suggestion
// NFTService contains all common interactions for NFTs
```

File: protobufs/account/service.proto [hunk 3]
```diff
@@ -11,7 +11,7 @@ import "google/api/annotations.proto";
 import "google/protobuf/empty.proto";
 import "protoc-gen-swagger/options/annotations.proto";
 
-// AccountService allows to manage accounts in the node
+// AccountService allows for the management of accounts in the node
```
Comment (vedhavyas) [inline protobufs/account/service.proto]:
```suggestion
// AccountService allows account management on the Node.
```

File: api/service.go (hunk 1 above)
Comment (charleenfei) [reply api/service.go]: yea, I agree. But I also can't think of a better way :(

**Pull request merged**